session/common.go

```diff
@@ -308,6 +310,15 @@ func (x *commonData) SetAuthKey(key neofscrypto.PublicKey) {
 	x.authKey = x.authKey[:key.Encode(x.authKey)]
 }
 
+// SetIssuer allows to set issuer before Sign call.
+// Using this setter at you own responsibility to ensure the ID matches ID of the signer, which will be used in Sing call.
```

Comment:
`s/Sing/Sign/`, though. Maybe even
`Using this method is not required when Sign is used (issuer will be derived from the signer automatically). When using it please ensure that the token is signed with the same signer as the issuer passed here.`

Reply:
Sure, typo. Updated to this text